src/components/RequestDashboard/PatientSection.jsx

```diff
@@ -0,0 +1,23 @@
+import React, { memo } from 'react';
+
+import useStyles from './styles';
+import RequestBuilder from '../../containers/RequestBuilder';
+import { SettingsContext } from '../../containers/ContextProvider/SettingsProvider';
+
+const PatientSection = props => {
+  const classes = useStyles();
+  const [state, dispatch] = React.useContext(SettingsContext);
+  // TODO: Make request builder use react-hooks so
+  // we can get rid of this hacky shim
```

Comment:
```suggestion
  // TODO: Make request builder use react-hooks
```
no need to be so self-deprecating! 😃 

In any case, I have some WIP code for a functional component version of RequestBuilder, but it won't be ready to merge into this branch so soon. We can probably contemplate if we want to keep the unused "RequestBuilder.consoleLog" method and use useRef for the RequestBuilder.taskSuggestion method. There's also a state update (fhirPatientQuery) that seems to depend on other pieces of state existing, and using timeouts might not be reliable since JS doesn't always operate in a top-down thread-like manner- not sure if that should be in its own useEffect with an empty dependency array `[]` or `[request.id]`.